po/ta.po

```diff
@@ -0,0 +1,575 @@
+
+#: src/models/portals.js:158
+msgid "Camera"
+msgstr "கேமரா"
+
+#: src/models/portals.js:160
+msgid "Can record videos with your camera"
+msgstr "உங்கள் கேமரா மூலம் வீடியோக்களை பதிவு செய்யலாம்"
+
+#: src/models/portals.js:168
+msgid "Location"
+msgstr "இருப்பிடம்"
+
+#: src/models/portals.js:170
+msgid "Can access your location"
+msgstr "உங்கள் இருப்பிடத்தை அணுக முடியும்"
+
+#: src/models/portals.js:208
+msgid "List of resources selectively granted to the application"
+msgstr "பயன்பாட்டிற்கு தேர்ந்தெடுக்கப்பட்ட ஆதாரங்களின் பட்டியல்"
+
+#: src/models/portals.js:248 src/widgets/permissionEntryRow.js:58
+#: src/widgets/permissionSwitchRow.js:43
+msgid "Not supported by the installed version of Flatpak"
+msgstr "பிளாட்பாக் இன் நிறுவப்பட்ட பதிப்பால் ஆதரிக்கப்படவில்லை"
+
+#: src/models/portals.js:256
+msgid "Requires permission store version 2 or newer"
+msgstr "அனுமதி ஸ்டோர் பதிப்பு 2 அல்லது புதியது தேவை"
+
+#: src/models/portals.js:264
+msgid "Portal data has not been set up yet"
+msgstr "போர்டல் தரவு இன்னும் அமைக்கப்படவில்லை"
+
+#: src/models/shared.js:39
+msgid "Network"
+msgstr "வலைப்பின்னல்"
+
+#: src/models/shared.js:46
+msgid "Inter-process communications"
+msgstr "இடை-செயல்முறை தொடர்புகள்"
+
+#: src/models/shared.js:79
+msgid "List of subsystems shared with the host system"
+msgstr "ஹோஸ்ட் சிஸ்டத்துடன் பகிரப்பட்ட துணை அமைப்புகளின் பட்டியல்"
+
+#: src/models/sockets.js:37
+msgid "X11 windowing system"
+msgstr "X11 சாளர அமைப்பு"
+
+#: src/models/sockets.js:44
+msgid "Wayland windowing system"
+msgstr "வேலேண்ட் சாளர அமைப்பு"
+
+#: src/models/sockets.js:51
+msgid "Fallback to X11 windowing system"
+msgstr "X11 விண்டோயிங் சிஸ்டத்திற்கு வீழ்ச்சி திரும்பவும்"
+
+#: src/models/sockets.js:58
+msgid "PulseAudio sound server"
+msgstr "பல்ஸ் ஆடியோ ஒலி சேவையகம்"
+
+#: src/models/sockets.js:65
+msgid "D-Bus session bus"
+msgstr "D-Bus அமர்வு பேருந்து"
+
+#: src/models/sockets.js:72
+msgid "D-Bus system bus"
+msgstr "D-Bus சிஸ்டம் பஸ்"
+
+#: src/models/sockets.js:79
+msgid "Secure Shell agent"
+msgstr "பாதுகாப்பான ஷெல் ஏஜென்ட்"
+
+#: src/models/sockets.js:86
+msgid "Smart cards"
+msgstr "ஸ்மார்ட் கார்டுகள்"
+
+#: src/models/sockets.js:93
+msgid "Printing system"
+msgstr "அச்சிடும் அமைப்பு"
+
+#: src/models/sockets.js:118
+msgid "List of well-known sockets available in the sandbox"
+msgstr "சாண்ட்பாக்ஸில் நன்கு அறியப்பட்ட சாக்கெட்டுகளின் பட்டியல்"
+
+#: src/models/variables.js:39
+msgid "Variables"
+msgstr "மாறிகள்"
+
+#: src/models/variables.js:42
+msgid "e.g. GTK_DEBUG=interactive"
+msgstr "எ.கா. GTK_DEBUG=interactive"
+
+#: src/models/variables.js:72
+msgid "List of variables exported to the application"
+msgstr "பயன்பாட்டிற்கு ஏற்றுமதி செய்யப்பட்ட மாறிகளின் பட்டியல்"
+
+#: src/widgets/aboutDialog.js:38
+msgid "Contributions by"
+msgstr "மூலம் பங்களிப்புகள்"
+
+#. TRANSLATORS: <full-month-name> <day-of-month>, <year-with-century>
+#: src/widgets/appInfoViewer.js:61
+msgid "%B %e, %Y"
+msgstr "%B %e, %Y"
```

Comment:
Regarding this string is this fine or should I make it into `<full-month-name> <day-of-month>, <year-with-century>` 's Tamil equivalent?